### DISCUSSION TITLE
Infinite scroll or Pagination

### DISCUSSION THREAD

**Discussion by obsidienne** [puemos/radiopush#31] [Ideas]
Hi @puemos,

Actually, we use infinite scroll to find posts. I'm not sure it's the best way to look throught posts.
For me, a pagination seems a better way to do it because, it

- allows a user to go the the seven page directly to find all posts
- decreases memory on the client side

If we stay with infinite scroll, may be we should swift to one of those two solution
- [Intersection Observer](https://developer.mozilla.org/fr/docs/Web/API/Intersection_Observer_API)
- A simple button (like google on mobil)

what do you think ?

**Comment by puemos** [comment]
I think there is no need to jump directly to page 7, the experience should resemble a Twitter feed.

At the moment the app is using a very simple mechanism to sense that the page is close to the end and then send an event to the page.
1. https://github.com/puemos/radiopush/blob/main/assets/js/hooks/InfiniteScroll.js
2. https://github.com/puemos/radiopush/blob/main/lib/radiopush_web/live/home_live/home_page.ex#L364

I do like the suggestion to replace it with a simple button, the whole feature will become simpler.

Do you have any thoughts on that? maybe you see the experience go in a direction of a blog?

**Reply by obsidienne** [reply]
The issue is that some of my channels have more than 200 posts. 
How can a new user look through all those posts. How can a user remember  certain posts. 

One solution is to have some kind of pagination
Another one is to have automatic channels (the posts I reacted to, posts from septembre in the following channels, a combination of everything,...)
The last is to send back to spotify the posts selected


As I'm writing, i will need automatic channel to create a top10

What do you think ?

**Comment by puemos** [comment]
Let's move this discussion to the new repo
